packages/components/src/internal/components/assay/AssayImportSubMenuItem.tsx

```diff
@@ -73,11 +54,8 @@ export const AssayImportSubMenuItemImpl: FC<Props & InjectedAssayModel> = props
     }
 
     let selectedCount = -1;
-
     if (queryModel !== undefined) {
         selectedCount = queryModel.selections?.size ?? -1;
-    } else if (model) {
-        selectedCount = model.selectedIds.size;
     }
 
     const overlayMessage =
```

Comment:
Remove `model: undefined` from the `menuProps`.

Reply:
good catch